openwisp_notifications/tests/test_notifications.py

```diff
@@ -944,6 +944,22 @@ def test_notification_for_unverified_email(self):
         # we don't send emails to unverified email addresses
         self.assertEqual(len(mail.outbox), 0)
 
+    @patch('openwisp_notifications.tasks.batch_email_notification.apply_async')
+    def test_batch_email_notification(self, mock_send_email):
```

Comment:
I would like to see some more test cases, testing different scenarios (sending batch email or single email). How the email template renders for 5+ notifications (description is absent) and 10 notifications (call to action button is present).